app/src/pages/inside/stepPage/modals/linkIssueModal/linkIssueModal.jsx

```diff
@@ -110,17 +131,33 @@ export class LinkIssueModal extends Component {
     },
   };
 
+  isBulkOperation = this.props.data.items.length > 1;
+
   constructor(props) {
     super(props);
-    const { namedBtsIntegrations, userId } = props;
+    const {
+      namedBtsIntegrations,
+      userId,
+      data: { items },
+    } = props;
     const { pluginName, integration } = getDefaultIssueModalConfig(namedBtsIntegrations, userId);
+    const currentItems = this.isBulkOperation
+      ? items.map((item) => {
+          return { ...item, itemId: item.id };
+        })
+      : items;
 
     this.props.initialize({
       issues: [{}],
     });
     this.state = {
       pluginName,
       integrationId: integration.id,
+      loading: false,
+      modalState: {
```

Comment:
Are you really need to wrap `testItems` and `selectedItems` into `modalState` object? Do you really need this complexity?